scripts/sommelier

```diff
@@ -151,7 +151,17 @@ install_app() {
     EXE_NAME=$(basename "${LOCAL_INSTALL_EXE}")
     yes | yad --progress --title="Installing ${EXE_NAME}" --progress-text= --width=400 --center --no-buttons --auto-close --auto-kill --on-top --pulsate &
     INSTALL_YAD_PID="$!"
-    "${WINELOADER}" "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"
+    FILE_END="${EXE_NAME##*.}"
+    FILE_TYPE="${FILE_END,,}" # force lowercase
+    if [[ "$FILE_TYPE" == 'exe' ]]; then
+        "${WINELOADER}" "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"
+    elif [[ "$FILE_TYPE" == 'msi' ]]; then
+        "${MSIEXEC}" /i "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"
+    else
+        echo "ERROR: Unsupported file type: $FILE_TYPE"
+        kill "$INSTALL_YAD_PID"
```

Comment:
I would instead print a WARNING here, and run the original logic (`"${WINELOADER}" "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"`). This way, we're sure we don't break existing scripts who use files with other extensions. The warning can say something like:

> `WARNING: Unsupported file type: $FILE_TYPE. This feature is deprecated, please contact the sommelier-core devs for more info.`

Reply:
I've corrected this too.